BookPlayer/Player/PlayerViewController.swift

```diff
@@ -185,7 +185,7 @@ class PlayerViewController: UIViewController, UIGestureRecognizerDelegate {
             },
             onEnd: { (_ cancelled: Bool) -> Void in
                 if !cancelled {
-                    PlayerManager.shared.stop()
+                    PlayerManager.shared.pause()
```

Comment:
thoughts?

Reply:
Totally for it. Given our current interaction model, `stop()` is used only when a book is about to be deleted. Or when it's paused and the mini player is dismissed. Pausing makes much more sense for the timer :)